src/components/Tetris.jsx

```diff
@@ -113,62 +116,67 @@ const Tetris = () => {
 		setTogglePause(true)
 	}
 
-	const move = ({keyCode}) => {
-	
+	const move = ({ keyCode }) => {
+
 		if (!gameOver) {
 			// left arrow
-			if (keyCode === 37 ) {
+			if (keyCode === 37) {
 				movePlayer(-1);
-			// right arrow
+				// right arrow
 			} else if (keyCode === 39) {
 				movePlayer(1);
-			// down arrow
+				// down arrow
 			} else if (keyCode === 40) {
 				dropPlayer();
-				
-            // up arrow      
-			}else if (keyCode === 38) {
-                playerRotate(stage, 1)
-            }else if (keyCode === 32) {
-				playerRotate(stage,1)
+
+				// up arrow      
+			} else if (keyCode === 38) {
+				playerRotate(stage, 1)
+			} else if (keyCode === 32) {
+				playerRotate(stage, 1)
 			}
 		}
 	}
 
-
-
+	// useEffect changes some states depending on the current screen width:
+	useEffect(() => {
+		mobileView ? setBtnText('Start') : setBtnText('Start Game');
+	}, [mobileView])
```

Comment:
just a short question to this one, why are you using a useEffect for that?

Reply:
to listen to changes of the mobileView variable and change the respective state while causing a re-render